modules/ROOT/pages/vpc-tutorial.adoc

```diff
@@ -25,7 +25,7 @@ In this case, the VPC is bound to `US East (Virginia)`.
 +
 [NOTE]
 --
-All VPCs need to be associated to a CloudHub region.
+All VPCs must be associated to a CloudHub region.
```

Comment:
associated with instead of to?

Reply:
Yes, but I think I fixed this in a different branch.